docs/content/mixin-dev-guide/commands.md

```diff
@@ -13,20 +13,18 @@ you are writing a mixin in Go, we strongly recommend starting from the template.
 
 **Required Commands**
 
-* [build](#build)
-* [schema](#schema)
-* [install](#install)
-* [upgrade](#upgrade)
-* [uninstall](#uninstall)
-* [version](#version)
```

Comment:
did you mean to get rid of this?

Reply:
Aahhh now I see what you mean 😄 
Yes it is expected, the section is not present on the website